index.js

```diff
@@ -0,0 +1,35 @@
+const hazyCalculator = (calculationSteps) => {
+  let operation
+
+  // Find Operation
+  for (const item of calculationSteps) {
+    if (item === '*' || item === '+' || item === '-' || item === '/') {
+      operation = item
+    }
+  }
+
+  // Filter or undefineds, empty strings, or NaN's
+  let filteredArr = calculationSteps.filter((item) => {
+    return !isNaN(Number(item)) && item !== ''
```

Comment:
interesting you do isNaN check on the converted value (Number) instead of the raw value of the array.